tensorflow/core/framework/device_factory.h

```diff
@@ -89,8 +90,10 @@ class DeviceFactory {
   // REGISTER_LOCAL_DEVICE_FACTORY to see the existing priorities used
   // for built-in devices.
   static int32 DevicePriority(const std::string& device_type);
-};
 
+  // Return the "subdevice_type" string for a "device_type string".
```

Comment:
Why are `subdevice_type` and `"device_type string` in quotes?

Reply:
Ok, I removed it. Originally I saw DevicePriority's comment put the device_type in the quotes. Anyway, seems I wrongly put the string in the quotes. Thanks for the reminder.
```
  // Return the device priority number for a "device_type" string.
  //
  // Higher number implies higher priority.
```